asf-site-src/source/documentation/in-progress/precommit-advanced.html.md

```diff
@@ -196,7 +196,7 @@ The `add_test` function is used to activate the standard tests.  Additional plug
 
 ## Module & Profile Determination
 
-Once the tests are determined, it is now time to pick which [modules](precommit-glossary.md#genericoutside-definitions) should get used.  That's the job of the `personality_modules` function.
+Once the tests are determined, it is now time to pick which [modules](../precommit-glossary#genericoutside-definitions) should get used.  That's the job of the `personality_modules` function.
```

Comment:
Why the `..` ? These paths look correct to me.

Reply:
The path is not correct in the generated docs. For example, in Yetus 0.11.1, the link is "https://yetus.apache.org/documentation/0.11.1/precommit-advanced/precommit-glossary.md#genericoutside-definitions" and I want to change it to "https://yetus.apache.org/documentation/0.11.1/precommit-glossary#genericoutside-definitions". That's why `..` is needed.

Reply:
Seems odd. Maybe it's better to configure the static site generator properly instead? I'm not familiar with how this is built.

Reply:
It's configured correctly. It makes links and URLs effectively directories instead of files for a cleaner look when users pass things around.

Reply:
Okay, so the links here are incorrect according to the overall site configuration. Thanks for explaining. You guys have a link to where site generation happens? I see we have a branch `asf-site`, but the link from https://github.com/apache/yetus/blob/master/asf-site-src/source/contribute.html.md to "Maintaining the Yetus Website" is also dead. Thanks.

Reply:
Oh, disregard that last query. The link is designed to be browsed from https://yetus.apache.org/contribute/website/, not from the source repo.